src/api/helpers/get-application-by-reference-number/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+
+/**
+ * getApplicationByReferenceNumber
+ * get ids of application sections by reference number
+ * returns application section ids or if not found, returns null
+ * @param {Number} referenceNumber
+ * @param {Context} context
+ * @returns {Application} application section ids or null
+ */
+const getApplicationByReferenceNumber = async (referenceNumber: number, context: Context): Application => {
+  const applications = (await context.db.Application.findMany({
+    where: {
+      referenceNumber: { equals: referenceNumber },
+    },
+  })) as Application;
+
+  if (applications.length) {
```

Comment:
`applications?.length` for safety?

Reply:
Probably won't need this if there's a try/catch handler, but can't see any harm in adding.